### DISCUSSION TITLE
The chart and app version are inconsistent

### DISCUSSION THREAD

**Discussion by MartinLau7** [kedacore/charts#571] [Q&A]
I found app Version 2.9.2 after installing Keda 2.9.3. Is this correct? The chart and app version are inconsistent

<img width="764" alt="image" src="https://github.com/kedacore/charts/assets/5416852/d49d0dd2-ec06-4dbf-9fe0-91a32bf9b225">

## Specifications

  - **KEDA Version:** *2.9.3*
  - **Platform & Version:** *Please elaborate*
  - **Kubernetes Version:** *Please elaborate*
  - **Scaler(s):** *Please elaborate*

**Comment by zroubalik** [comment]
That is intentional, sometimes we update only charts and not the core KEDA.
